content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -165,22 +163,19 @@ await prisma.$transaction(
 )
 ```
 
-### Interactive transactions (in Preview)
+### Interactive transactions
 
 Sometimes you need more control over what queries execute within a transaction. Interactive transactions are meant to provide you with an escape hatch.
 
-**Note**: Interactive Transactions with the [Data Proxy](/data-platform/data-proxy#providing-the-connection-string-to-prisma-migrate) are in Preview in version 4.6.0.
+<Admonition>
 
-To enable interactive transactions, add `interactiveTransactions` to the generator of your Prisma Schema:
+**Note**<br /><br />
 
-```prisma
-generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["interactiveTransactions"]
-}
-```
+Interactive Transactions with the [Data Proxy](/data-platform/data-proxy#providing-the-connection-string-to-prisma-migrate) are in General Avaialbility since version 4.7.0.
```

Comment:
```suggestion
Interactive Transactions with the [Data Proxy](/data-platform/data-proxy#providing-the-connection-string-to-prisma-migrate) are in General Availability from version 4.7.0.
```

Corrected a typo; also, I think "from" is a bit more natural than "since" in this context